src/ExpansionCard/ExpansionCard.interface.ts

```diff
@@ -0,0 +1,19 @@
+import { Modal } from '@delangle/use-modal'
+
+import { CardProps } from '../Card'
+
+export interface ExpansionCardProps
+  extends Omit<CardProps, 'title' | 'children'> {
+  children?: React.ReactNode
+  defaultOpen?: boolean
+  description?: string | React.ReactNode
+  disabled?: boolean
+  error?: boolean
```

Comment:
We should avoid usage of boolean for these styles in the future.
I think something like `style: 'error' | 'warning'` would be much better

Reply:
@jean9696 Here I am just passing the props of `ExpansionPanelItem`. I am not sure that I want to make the change for this component here